bundle-size/pm.sh

```diff
@@ -17,27 +17,23 @@ install_pnpm_if_not_present() {
 # Detect the package manager
 if [ -f yarn.lock ]; then
 	pm="yarn"
-elif [ -f pnpm-lock.yaml ]; then
-	install_pnpm_if_not_present
-	pm="pnpm"
 elif [ -f package-lock.json ]; then
 	pm="npm"
 else
-	echo "No recognized package manager found in this project."
-	exit 1
+	echo "Defaulting to pnpm for install command."
+	install_pnpm_if_not_present
+	pm="pnpm"
 fi
```

Comment:
In IRM monorepo we have `pnpm-lock` in the root but the action itself runs in subdirectory, e.g. `packages/@plugins/grafana-oncall-app`. We could potentially tackle that differently but a simple solution for now would be to default to `pnpm` in case the `yarn.lock` and `package-lock.json` is not found in the current dir. 

Unless you have a better idea for this @jackw :)